app/src/components/buttons/filterButton/filterButton.jsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { Popover } from '@reportportal/ui-kit';
+import Parser from 'html-react-parser';
+import filterIcon from 'common/img/newIcons/filters-outline-inline.svg';
+import filterFilledIcon from 'common/img/newIcons/filter-filled-inline.svg';
+import { FilterContent } from './filterContent';
+import styles from './filterButton.scss';
+
+const cx = classNames.bind(styles);
+
+export const FilterButton = ({
```

Comment:
Let's agree on reusable components and their location.
I suggest to store only not connected (to Redux and other third-parties) components in the components/{buttons, inputs, etc.}. There are some kind of primitives. Moreover, most of them will be removed gradually within switching to the new designs. So, let's create a new directory for such stuff, or move it to the `components/main`.

Reply:
Done.